samples/drivers/CAN/src/main.c

```diff
@@ -220,7 +220,7 @@ void main(void)
 	}
 
 	ret = gpio_pin_configure(led_gpio_dev, DT_ALIAS_LED0_GPIOS_PIN,
-				 GPIO_DIR_OUT);
+				 GPIO_OUTPUT_HIGH | DT_ALIAS_LED0_GPIOS_FLAGS);
```

Comment:
I think this should be:
https://github.com/zephyrproject-rtos/zephyr/blob/2c6af1f36d1f860b207a0083d134a3d329754e54/samples/basic/button/src/main.c#L66-L67

Reply:
Depending on what this command intended to do (initialize the LED off or initialize the LED on) you can't express it correctly in devicetree now.  We'd need `GPIO_OUTPUT_INACTIVE` or `GPIO_OUTPUT_ACTIVE` which map to either `GPIO_OUTPUT_HIGH` or `GPIO_OUTPUT_LOW` depending on the active level.  Nobody's implemented that yet.  Just setting `GPIO_OUTPUT` would leave the state indeterminate (or the reset value).